### DISCUSSION TITLE
Adopt Tailwind for Quasar v3

### DISCUSSION THREAD

**Discussion by BryanHuntNV** [quasarframework/quasar#16253] [Ideas / Proposals]
I have seen several people ask about using Tailwind with Quasar.  I would love to use Tailwind with Quasar, but my team wants to stick with the built-in CSS framework.  Major versions are for significant change right?  What if Quasar adopted Tailwind as its built-in CSS framework for v3?

**Comment by dongwa** [comment]
I see that there is a Dart available, but no progress has been made as of now.
https://github.com/orgs/quasarframework/projects/2?query=is%3Aopen+sort%3Aupdated-desc&pane=issue&itemId=1451670
